next.config.js

```diff
@@ -7,6 +7,10 @@ module.exports = withTypescript(
     withCSS({
       useFileSystemPublicRoutes: false,
       assetPrefix: process.env.BASE_URL || '',
+      publicRuntimeConfig: {
+        uidName: 'email',
+        uidArticle: 'an',
```

Comment:
It would be easier to configure this using something like:
    uidName: process.env.UID_NAME || 'email',
    uidArticle: process.env.UID_ARTICLE || 'an',

Reply:
Personally, I think it could get tedious to have to type `UID_NAME=foo UID_ARTICLE=bar ... npm run build` every time you want to run or build code. I'd prefer that config like this exist somewhere on disk. Is there anything particularly attractive about environment variables for you? How would you feel about something like a separate `config.js` file that doesn't get checked into the repo, and then mirroring values from that (if it exists) here?

Reply:
A place to put local configuration that doesn't edit files in the repo is exactly what I'd like. :)
I believe the standard place to write local configuration is in a .env file, usually using the dotenv package.
As far as just using plain old environment variables goes, we just drop in 'export VAR=UBC' into .bash_profile and done... We're already doing this with Queue's existing use of environment variables like JWT_SECRET.

Reply:
I'll add support for a `.env` file for configuration

Reply:
Actually @winstan, let's just add that in #297.